tests/integration/targets/aci_fabric_interface_policy_group/tasks/main.yml

```diff
@@ -0,0 +1,327 @@
+# Test code for the ACI modules
+# Copyright: (c) 2023, Sabari Jaganathan (@sajagana)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
+    msg: "Please define the following variables: aci_hostname, aci_username and aci_password."
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+# SET VARS
+- name: Set vars
+  set_fact:
+    aci_info: &aci_info
+      host: "{{ aci_hostname }}"
+      username: "{{ aci_username }}"
+      password: "{{ aci_password }}"
+      validate_certs: "{{ aci_validate_certs | default(false) }}"
+      use_ssl: "{{ aci_use_ssl | default(true) }}"
+      use_proxy: "{{ aci_use_proxy | default(true) }}"
+      output_level: '{{ aci_output_level | default("info") }}'
+
+- name: Query system information
+  cisco.aci.aci_system:
+    <<: *aci_info
+    id: 1
+    state: query
+  register: version
+
+- name: Ensure leaf_policy_group does not exist
+  aci_fabric_interface_policy_group:
+    <<: *aci_info
+    name: leaf_policy_group
+    type: leaf
+    state: absent
+
+- name: Ensure spine_policy_group does not exist
+  aci_fabric_interface_policy_group:
+    <<: *aci_info
+    name: spine_policy_group
+    type: spine
+    state: absent
+
+- name: Add a leaf fabric interface policy group with check mode
+  aci_fabric_interface_policy_group: &cm_leaf_policy_group_present
+    <<: *aci_info
+    name: leaf_policy_group
+    type: leaf
+    descr: leaf_policy_group created
+    state: present
+  check_mode: true
+  register: cm_leaf_policy_group_present
+
+- name: Add a leaf fabric interface policy group with normal mode
+  aci_fabric_interface_policy_group: &nm_leaf_policy_group_present
+    <<: *cm_leaf_policy_group_present
+    dwdm_policy: default
+    link_level_policy: default
+    link_flap_policy: default
+    l3_interface_policy: default
+    macsec_policy: default
+    monitoring_policy: default
+  register: nm_leaf_policy_group_present
+
+- name: Add a leaf fabric interface policy group with normal mode again
+  aci_fabric_interface_policy_group:
+    <<: *nm_leaf_policy_group_present
+  register: nm_leaf_policy_group_present_again
+
+- name: Add a spine fabric interface policy group with check mode
+  aci_fabric_interface_policy_group: &cm_spine_policy_group_present
+    <<: *aci_info
+    name: spine_policy_group
+    type: spine
+    descr: spine_policy_group created
+    dwdm_policy: default
+    link_level_policy: default
+    link_flap_policy: default
+    l3_interface_policy: default
+    macsec_policy: default
+    monitoring_policy: default
+    state: present
+  check_mode: true
+  register: cm_spine_policy_group_present
+
+- name: Execute tasks only for ACI v6+
+  when: version.current.0.topSystem.attributes.version is version('6', '>=')
```

Comment:
https://173.36.219.73/, [cn-dmz-apic-m1-07-v32] fails with unknown class so the version needs to be more specific. This is version Version 6.0(1g). 173.36.219.70 version Version 6.0(2h) does work properly. Probably also best to update in documentation.